app/models/provider.rb

```diff
@@ -205,32 +214,15 @@ def external_contact_info
       region_code
       telephone
       email
+      website
     ]
 
-    if enrichments.last
-      enrichments.last.attributes.slice(*(attribute_names + %w[website]))
-    else
-      attributes.slice(*attribute_names).merge("website" => url)
-    end
+    attributes.slice(*attribute_names)
   end
 
+  # NOTE: This can be removed, it should not be in use any more
   def content_status
```

Comment:
Any reason why we aren't removing it?

Reply:
https://trello.com/c/76g6Jm4S/2386-remove-contentstatus-and-lastpublishedat-from-provider-both-mcbe-and-mcfe